connector/connect/server/src/main/scala/org/apache/spark/sql/connect/service/SessionHolder.scala

```diff
@@ -45,6 +49,15 @@ case class SessionHolder(userId: String, sessionId: String, session: SparkSessio
   private val executions: ConcurrentMap[String, ExecuteHolder] =
     new ConcurrentHashMap[String, ExecuteHolder]()
 
+  // The cache that maps an error id to a throwable. The throwable in cache is independent to
+  // each other.
+  val errorIdToError = CacheBuilder
```

Comment:
Why is this public? I'd expect style checks to complain about a missing type annotation.

Reply:
Good catch, it shouldn't...